apps/worker/src/apps/convertkit/index.ts

```diff
@@ -0,0 +1,67 @@
+import { IEventData } from "../../types";
+import { httpClient } from "../../../utils";
+
+interface IConvertKitData extends IEventData {
+  userId: string;
+  actionSlug: string;
+  apiKeys: {
+    additionalData: any;
+    accessToken: string;
+  };
+  app: {
+    slug: string;
+    name: string;
+    id: string;
+    status: string;
+    appConfig: {
+      baseUrl: string;
+      [key: string]: any;
+    };
+  };
+}
+
+interface IAddSubscriberToAForm {
+  accessToken: string;
+  formId: string;
+  baseUrl: string;
+  data: {
+    [key: string]: any;
+  };
+}
+
+const addSubscriberToAForm = async ({
+  accessToken,
+  data,
+  formId,
+  baseUrl = `https://api.convertkit.com/v3/forms/${formId}`,
```

Comment:
baseUrl must not have anything dynamic variable

just make it as `https://api.convertkit.com/v3/forms`

for endpoint for `${baseUrl}/{formId}/subscribe`

Reply:
changes has done